mapdata/WarcraftLegacies/AbilityData/Skin/925905217.json

```diff
@@ -14,7 +14,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "Gives 50% additional armor to nearby friendly units."
+      "Value": "Gives 50% additional base armor to nearby friendly units."
```

Comment:
I feel like this probably doesn't say what you intended it to - this says it increases _base_ armor, which it doesn't. I'd just leave it as is; vanilla WC3 tooltips (ike Endurance Aura) don't explicitly say they increase by a percentage of the base and I don't it adds anything for us to either.

Reply:
This was a request from Techno, as a unit with 0 base armor but upgrades resulting in 2 armour will not gain any benefit from this aura.

Reply:
```suggestion
      "Value": "Increases the armor of nearby friendly units by 50% of their base armor, not including any upgrades."
```